web-app/src/components/TakenLeave/index.jsx

```diff
@@ -1,29 +1,35 @@
 import React from 'react';
+import { PropTypes as PT } from 'prop-types';
 import styles from './style.css';
 import container from './container';
 
 const TakenLeave = (props) => {
 
-    const formatDate = (date) => {
-        const definedDate = new Date(date);
-        const year = definedDate.getFullYear();
-        const month = definedDate.toLocaleString("en-us", { month: "long"});
-        const day = definedDate.getDate();
+  const formatDate = (date) => {
+    const definedDate = new Date(date);
+    const year = definedDate.getFullYear();
+    const month = definedDate.toLocaleString('en-us', { month: 'long'});
+    const day = definedDate.getDate();
 
-        return year + '-' + month + '-' + day;
-    }
+    return year + '-' + month + '-' + day;
+  };
 
-    return (
-      <div>
-          {
-              props.takenHolidays.map(holiday => {
-                  return <span className={styles.TextStyle}>
-                      {formatDate(holiday.date)}
-                  </span>
-              })
-          }
-      </div>
-    );
+  return (
+    <div>
+      {
+        props.takenHolidays.map(holiday => {
```

Comment:
What is props here? What exists in that object? Would be nice to spread it then take what we need and use a better naming convention.